cmd/ocsp-updater/main.go

```diff
@@ -145,6 +144,7 @@ func main() {
 		serialSuffixes,
 		ogc,
 		// Necessary evil for now
+		// TODO(XXX): Fix this, or file a bug to fix it later.
```

Comment:
I assume you meant to fill in XXX before merging?

Reply:
Filed https://github.com/letsencrypt/boulder/issues/5904 for this.